assets/model_monitoring/components/prediction_drift/prediction_drift_signal_monitor/spec.yaml

```diff
@@ -104,7 +104,7 @@ jobs:
       type: aml_token
   compute_histogram_buckets:
     type: spark
-    component: azureml://registries/azureml/components/model_monitor_compute_histogram_buckets/versions/0.3.5
+    component: azureml://registries/azureml/components/model_monitor_compute_histogram_buckets/versions/0.3.6
```

Comment:
you also need to upgrade the version for model_monitor_compute_histogram_buckets component